index.html

```diff
@@ -0,0 +1,545 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Harshil's Component Library</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="shortcut icon" type="image/jpg" href="/images/favicon.ico" />
+    <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+    <div class="first">
+        <h2><span class="span">Harshil's Component Library :</span> A component library for all your needs</h2>
+    </div>
+    <section id="page">
+        <nav class="nav">
+            <ul>
+                <a href="#avatar-id">
+                    <li> Avatar</li>
+                </a>
```

Comment:
```  <li><a></a></li> ```
I think the above one is the correct way of defining the list item and anchor tags.